packages/backend/src/projectAdapters/projectAdapter.ts

```diff
@@ -25,11 +25,15 @@ export const projectAdapterFromConfig = async (
         warehouseClientFromCredentials(warehouseCredentials);
     const configType = config.type;
     Logger.debug(`Initialize project adaptor of type ${configType}`);
+
     switch (config.type) {
         case DbtProjectType.DBT:
             return new DbtLocalCredentialsProjectAdapter({
                 warehouseClient,
-                projectDir: config.project_dir || '/usr/app/dbt',
+                projectDir:
+                    config.project_dir ||
+                    process.env.DBT_PROJECT_DIR ||
```

Comment:
backend ignored env var so I added it here..

![CleanShot 2022-10-06 at 10 53 57@2x](https://user-images.githubusercontent.com/962095/194234350-0b54bc5e-58de-471c-8fc7-b1c6625c0716.png)